doc/source/redis-memory-management.rst

```diff
@@ -0,0 +1,97 @@
+Redis Memory Management (EXPERIMENTAL)
+======================================
+
+Ray stores metadata associated with tasks and objects in one or more Redis
+servers, as described in :doc:`internals-overview`.  Applications that are
```

Comment:
Ok, I just built the documentation locally and this seems to work. However, it seems like we're using two different link styles to accomplish the same thing.

Here we have

```
:doc:`internals-overview`
```

and below we have

```
<installation.html#building-ray-from-source>`__
```

Why not just stick with one style? Or am I missing something?

Reply:
Changed to the same style now